docs/source/scylla_cluster_autoscaler_crd.md

```diff
@@ -0,0 +1,104 @@
+# Scylla Cluster Autoscaler CRD
+
+Scylla clusters can be created and configuring using the 'scyllaclusterautoscalers.scylla.scylladb.com' custom resource definition (CRD).
+
+Please refer to the the [user guide walk-through](generic.md) for deployment instructions.
+This page will explain all the available configuration options on the Scylla Cluster Autoscaler CRD.
+
+## Sample
+
+```yaml
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: ScyllaClusterAutoscaler
+metadata:
+  name: example-sca
+spec:
+  targetRef:
+    namespace: scylla
+    name: example-cluster
+  updatePolicy:
+    updateMode: Auto
+  scalingPolicy:
+    datacenters:
+    - name: us-east-1
+      racks:
+      - name: us-east-1a
+        rules:
+        - name: cpu utilization horizontal up
+          priority: 1
+          expression: 'avg(scylla_reactor_utilization{scylla_cluster="example-cluster", scylla_datacenter="us-east-1", scylla_rack="us-east-1a"}) > bool 70'
+          mode: Horizontal
+          for: 10m
+          step: 30s
+          factor: 2
+        - name: cpu utilization horizontal down
+          priority: 1
+          expression: 'avg(scylla_reactor_utilization{scylla_cluster="example-cluster", scylla_datacenter="us-east-1", scylla_rack="us-east-1a"}) < bool 10'
+          mode: Horizontal
+          for: 10m
+          step: 30s
+          factor: 0.5
+        memberPolicy:
+          minAllowed: 1
+          maxAllowed: 5
+        resourcePolicy:
+          minAllowedCpu: 1
+          maxAllowedCpu: 2
+          controlledValues: Requests
+status:
+  lastApplied: 2021-04-14T11:54:17Z
+  lastUpdated: 2021-04-14T16:43:22Z
+  updateStatus: Ok
+  recommendations:
+    datacenterRecommendations:
+      - name: us-east-1
+        rackRecommendations:
+          - name: us-east-1a
+            members: 2
+            resources:
+              limits:
+                cpu:     2
+                memory:  30Gi
+              requests:
+                cpu:     2
+                memory:  30Gi
+```
+
+## Autoscaler Settings
+
```

Comment:
Inlcude information about the type for each of the fields

Reply:
Added in following format:
`*maxAllowedCpu: [Quantity](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity), optional field. Maximum Rack's CPU resource quantity. SCA won't scale CPU resource above this quantity.`
  `* controlledValues: Enum, optional field. Can be set to either "Requests" or "RequestsAndLimits" (default "RequestsAndLimits"). Which resource values should be scaled.`